static/icons/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "TBA",
```

Comment:
I'm not sure if Apple uses it, but it's a standard web app manifest thing. https://developer.chrome.com/extensions/manifest